hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/TestOMRequestUtils.java

```diff
@@ -439,6 +460,20 @@ public static void addBucketToDB(String volumeName, String bucketName,
     return metadataList;
   }
 
+  public static List< HddsProtos.KeyValue> getMetadataListV1() {
+    List<HddsProtos.KeyValue> metadataList = new ArrayList<>();
+    metadataList.add(HddsProtos.KeyValue.newBuilder().setKey("key1").setValue(
+            "value1").build());
+    metadataList.add(HddsProtos.KeyValue.newBuilder().setKey("key2").setValue(
+            "value2").build());
+    metadataList.add(HddsProtos.KeyValue.newBuilder().setKey(
+            OMConfigKeys.OZONE_OM_LAYOUT_VERSION).setValue(
+            OMConfigKeys.OZONE_OM_LAYOUT_VERSION_V1).build());
+    metadataList.add(HddsProtos.KeyValue.newBuilder().setKey(
+            OMConfigKeys.OZONE_OM_ENABLE_FILESYSTEM_PATHS).setValue(
+            "false").build());
```

Comment:
Not get this, why we pass L1 version but not enabled FILESYSTEM_PATHS here

Reply:
Like I explained in below comment, test is to verify the stored elements can be matched against bucketInfo and there is no relation with FSO enabled or not. If required I can make `OZONE_OM_ENABLE_FILESYSTEM_PATHS=true` value, should I change to true?

Reply:
Okay, get it.